package.json

```diff
@@ -57,6 +57,7 @@
     "passport-http-bearer": "^1.0.1",
     "pino-http": "^10.3.0",
     "stoppable": "^1.1.0",
+    "validator": "^13.12.0",
```

Comment:
issue: We could use [Zod](https://zod.dev/) for validation since we already have it as a dependency